nextstrain/cli/util.py

```diff
@@ -598,10 +598,12 @@ def runner_tests_ok(tests: RunnerTestResults) -> bool:
     return False not in [result for test, result in tests]
 
 
-def print_runner_tests(tests: RunnerTestResults):
+def stream_and_print_results(tests: RunnerTestResults) -> RunnerTestResults:
```

Comment:
I think "passthru" is the more relevant word than "stream": the input is (passed thru to) the output.

Also, "results" is left unqualified in the new function name. What results? Whose results?

```suggestion
def passthru_and_print_runner_tests(tests: RunnerTestResults) -> RunnerTestResults:
```

Reply:
Updated to `passthru_and_print_runner_tests`:

https://github.com/nextstrain/cli/blob/f6ddf6e139fd56aa63568955901c11786717e0fd/nextstrain/cli/util.py#L601

then changed to `print_and_check_runner_tests` for https://github.com/nextstrain/cli/pull/418#discussion_r2010958707